### PR TITLE
Update Solidity version example

### PR DESCRIPTION
**Pull request by pcowgill**

### CODE REVIEW

**Comment by pcowgill** [comment]
Is this good to merge? Thanks!

**Comment by alcuadrado** [comment]
Hey @pcowgill! Thanks for submitting this PR.

I'm a bit hesitant about changing 4/5 to 5/6 instead of just reword it to something that indicates that multiple versions can be used. What do you think?

**Comment by pcowgill** [comment]
@alcuadrado Whatever you think is best works for me. But I think some change from the status quo is warranted. Thanks!

**Comment by alcuadrado** [comment]
I added a comment directly to your branch. Will merge this when the Netlify check passes. 

Thanks @pcowgill
